### ISSUE TITLE
No start menu shortcut after installing from PPA

### ISSUE THREAD

**Issue by dwvisser** [quickgui/quickgui#12]
I installed from the PPA installation instructions in the README at https://github.com/wimpysworld/quickemu#quickgui-for-ubuntu, and I don't see a launcher shortcut. I *am* able to launch with the `quickgui` command from a terminal. I am using Ubuntu MATE 21.10:

```bash
➜  lsb_release --all
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 21.10
Release:	21.10
Codename:	impish

➜  uname -a
Linux octavia 5.13.0-21-generic #21-Ubuntu SMP Tue Oct 19 08:59:28 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux
```

**Comment by ymauray** [comment]
Well spotted. There is no launcher as of yet. Something I need to work on. Thanks for opening this issue!

**Comment by ymauray** [comment]
Fixed in v1.1.3
